lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -572,7 +572,7 @@ def main():
         http_timeout=dict(type='int'),
         load_path=dict(type='path'),
         name=dict(type='str', required=True),
-        nocache=dict(type='str', default=False),
+        nocache=dict(type='bool', default=False),
```

Comment:
Could you please update the `DOCUMENTATION` section to reflect this change? Except that LGTM.